bower.json

```diff
@@ -1,24 +1,26 @@
 {
   "name": "angular-css",
   "main": "./angular-css.js",
-  "version": "1.0.8",
+  "version": "1.0.9",
```

Comment:
Can you not bum the version? That way I can plan better what goes into what version.